### PR TITLE
changes in prm and astar

### LINKED ISSUE #25: Add plan function in PRM Path Planner

### PR DESCRIPTION
**Pull request by archit2604**
added plan function in prm.py according to the core.py template
made changes in astar.py to correct bugs
added test_astar_heuristic function which uses manhattan distance as heuristic
made changes in test_prm to use plan function

### CODE REVIEW

File: gennav/planners/graph_search_algorithms/astar.py [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
 
-try:
-    from queue import PriorityQueue, Queue
-except ImportError:
-    from Queue import PriorityQueue, Queue
+
+class Node:
+
+    # Initialize the class
+    def __init__(self, node, parent):
```
Comment (threewisemonkeys-as) [inline gennav/planners/graph_search_algorithms/astar.py:7]: Could you add docstring here

Comment (archit2604) [reply gennav/planners/graph_search_algorithms/astar.py:7]: Sure

**Comment by threewisemonkeys-as** [comment]
Could you try to add `Node` to `utils/common/py` with the other data structure. You would need to make it a little more general purpose though

**Comment by archit2604** [comment]
Ok

**Pull request merged**